sdk/confidentialledger/azure-confidentialledger/azure/confidentialledger/receiptverification/_serialization.py

```diff
@@ -0,0 +1,2125 @@
+# --------------------------------------------------------------------------
+#
```

Comment:
Why is this file here?

Reply:
I wasn't aware we were taking the distance from such a pattern. I think it does make sense to validate the structure of the JSON, its content is effectively validated by the algorithm implemented for receipt verification.

In any case, I understand this serialization code does not really have a use case to be here, so I removed it.